src/nvim/msgpack_rpc/remote_ui.c

```diff
@@ -66,8 +66,9 @@ static Object remote_ui_attach(uint64_t channel_id, uint64_t request_id,
       || args.items[2].type != kObjectTypeBoolean
       || args.items[0].data.integer <= 0 || args.items[1].data.integer <= 0) {
     api_set_error(error, Validation,
-                  _("Arguments must be a pair of positive integers "
-                    "representing the remote screen width/height"));
+                  _("Arguments must be a pair of positive integers and a "
+                    "boolean representing the remote screen width, height "
+                    "and whether to use RGB"));
```

Comment:
Sorry for the late response; I guess the small size of this change isn't attracting much attention. Perhaps it could be:
```
Arguments must be:
- remote screen width (integer >= 0)
- remote screen height (integer >= 0)
- whether to enable RGB (boolean)
```

`whether to enable RGB (boolean)` sounds a bit cryptic, I wonder if it means `whether to enable RGB capabilities (boolean)`.